_data/api-swagger.yaml

```diff
@@ -564,6 +576,10 @@ definitions:
             Must conform to E.164,
             [the international public telecommunication numbering plan](https://en.wikipedia.org/wiki/E.164),
             which requires the country calling code (e.g. +31).
+        SecureDownload:
```

Comment:
specify default value

Reply:
I suggest: default = false.

That takes away confusion. Where customers try to interpret a postback or response with this attribute missing they can still deduce expected behaviour of transaction